lib/vector-api-client/src/gql/components.rs

```diff
@@ -70,71 +70,53 @@ impl ComponentsSubscriptionExt for crate::SubscriptionClient {
 }
 
 impl components_query::ComponentsQueryComponentsEdgesNodeOn {
-    pub fn processed_events_total(&self) -> i64 {
+    pub fn received_bytes_total(&self) -> i64 {
+        // This is network bytes received, and only sources can receive events.
```

Comment:
This is out-of-scope of this PR, but in #17322 we are standardizing on the component estimated JSON byte size for received/sent event bytes. We can make that update later since this preserves the current behavior.

Reply:
That's fair.

IMO, showing the event bytes in, or event bytes out, for a source or sink, respectively, wouldn't make sense: the numbers can be vastly different in many cases. Obviously, for the bytes _out_ of a source, bytes _into_ a sink, or bytes in/out of a transform, it _has_ to be the event bytes: that's all we have.

The `top` UI is slightly cluttered as-is, but it might be nice/useful to think about somehow demarcating which bytes value we're using in the respective column, so that _all_ components can show a value, but it's clear that, oh, OK, yeah, I'm getting 10MB/s in over the wire to my source, and sending out 5MB/s, and events bytes for the remainder just to fill things out.